jenkins/Makefile

```diff
@@ -120,4 +120,4 @@ development.ubi.Dockerfile: development.Dockerfile.j2
 		> $@
 
 clean:
-	rm -f development.*.Dockerfile
+	rm -f $(shell cat .gitignore)
```

Comment:
I think this is arguably wrong, because of the semantics of `.gitignore`. I suspect we're assuming this is run inside `<gitroot>/jenkins` so we're only processing your new `.gitignore` entries and the `jenkins` directory currently isn't deep... however, technically, an unanchored `.gitignore` pattern matches anywhere in the subtree, meaning that `ci.*.Dockerfile` would match `./jenkins/ci.foo.Dockerfile` *and* `./jenkins/something/ci.bar.Dockerfile` so you'd be cleaning just a subset.

This is more equivalent to a `find -exec rm {} \+` although the mechanism of translating a `.gitignore` pattern into a `-name` is a bit more tricky than inclusion.

Like Webb, if for a completely different reason, I guess I'm willing to "hold my nose and move along" because I suspect the *current* local `.gitignore` will work in the current context... but the assumptions here bother me.